examples/hercules_input_example.yaml

```diff
@@ -0,0 +1,177 @@
+# Input YAML for Hercules simulation
```

Comment:
I love this, this is great!! I wonder if we could automate it in the future so it updates when we update models?